src/functional/iterator/builtins.py

```diff
@@ -136,3 +163,141 @@ def make_tuple(*args):
 @builtin_dispatch
 def tuple_get(*args):
     raise BackendNotSelectedError()
+
+
+# FIXME(ben): various of the following built-ins will shadow python built-ins!
+# We should find a good way around this...
```

Comment:
This is particularly annoying if you want to create a jupyter notebook which executes stencils, but also uses built-in functions in normal Python code. E.g., `min` and `max` are built-ins that won't work anymore if one just `from ....builtins import *`.